src/services/0_ExternToolsMS/logic/EtherpadClient.js

```diff
@@ -78,70 +77,70 @@ class EtherpadClient {
 		return rp(this.createSettings({
 			endpoint: 'createAuthorIfNotExistsFor',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetAuthor) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetAuthor) } );
 	}
 
 	createOrGetGroup(params) {
 		return rp(this.createSettings({
 			endpoint: 'createGroupIfNotExistsFor',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetGroup) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.createOrGetGroup) } );
 	}
 
 	getActiveSessions(params) {
 		return rp(this.createSettings({
 			endpoint: 'listSessionsOfAuthor',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.getActiveSessions) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.getActiveSessions) } );
 	}
 
 	createSession(params) {
 		return rp(this.createSettings({
 			endpoint: 'createSession',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => { this.handleEtherpadError(err, this.err.createSession) } );
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => { this.handleEtherpadError(err, this.err.createSession) } );
 	}
 
 	createOrGetGroupPad(params) {
-		if(params.oldPadId) {
-			let newPadId = `${params.groupID}$${params.padName}`;
-			let copyParams = {
+		if (params.oldPadId) {
+			const newPadId = `${params.groupID}$${params.padName}`;
+			const copyParams = {
 				sourceID: params.oldPadId,
 				destinationID: newPadId,
 			};
 			return rp(this.createSettings({
 				endpoint: 'copyPad',
 			}, copyParams))
-			.then((res) => {
-				let response = this.handleEtherpadResponse(res);
-				response.data = {
-					padID: newPadId
-				};
-				return response;
-			})
-			.catch((err) => { this.handleEtherpadError(err, this.err.copyOldPadToGroupPad) } );
+				.then((res) => {
+					const response = this.handleEtherpadResponse(res);
+					response.data = {
+						padID: newPadId,
+					};
+					return response;
+				})
+				.catch((err) => { this.handleEtherpadError(err, this.err.copyOldPadToGroupPad) } );
 		}
 		return rp(this.createSettings({
 			endpoint: 'createGroupPad',
 		}, params))
-		.then((res) => this.handleEtherpadResponse(res))
-		.catch((err) => {
-			// pad is already there, just return the constructed pad path
-			if(err.code === 409) {
-				return {
-					code: 0,
-					message: 'ok',
-					data: {
-						padID: `${params.groupID}$${params.padName}`
-					}
+			.then((res) => this.handleEtherpadResponse(res))
+			.catch((err) => {
```

Comment:
Codacy found an issue: [Expected to return a value at the end of arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5768037)